source/basic.tex

```diff
@@ -3897,7 +3897,7 @@
 floating-point types}.
 \indextext{floating-point type!implementation-defined}%
 \begin{note}
-This International Standard imposes no requirements on the accuracy of
```

Comment:
I feel this change is a regression, but I can't quite say why. I think it's clear to me that a Standard can make impositions, but it sounds odd that a document would do so.

Reply:
I think this is an improvement, for what it's worth. It clarifies that this document imposes no requirements, but other documents (for instance, other standards that your implementation might follow, such as ISO/IEC 60559, or vendor-specific compiler flags, may impose further requirements). The fact that this document is an International Standard is irrelevant here.

Reply:
I suppose that makes sense. It's probably just my lack of familiarity with the phrase, and I'll get used to it. I can see the consistency argument.